pkg/provisioner/ironic/ironic.go

```diff
@@ -1205,6 +1216,28 @@ func (p *ironicProvisioner) PowerOn() (result provisioner.Result, err error) {
 func (p *ironicProvisioner) PowerOff() (result provisioner.Result, err error) {
 	p.log.Info("ensuring host is powered off")
 
+	result, err = p.softPowerOff()
```

Comment:
The result values returned here are not actually used anywhere, so why bother returning them?

Reply:
These two values are returned as the result of the `PowerOff` function, I refer to the `PowerOn` and `PowerOff` functions, they will have these two return values.